how-we-work/oncall-responsibilities.md

```diff
@@ -58,8 +58,8 @@ The instructions to deploy `hosted-cellxgene` (a.k.a. cellxgene Explorer) can be
 
 #### Other responsibilities
 
-- If the secondary on-call engineer is new, please pair-program to help them onboard onto the process. Include them when you do deployment and point out various issues to watch out for (i.e. incoming bugs, Sentry, etc.) so that they get an understand of what the role entails.
-- Watch the #single-cell-ops, #single-cell-eng, and #single-cell-data-wrangling Slack channels for any questions or issues reported by any engineers, PM, or our Curation team. Please triage these issues, notify the right person, and if appropriate file an issue to keep track of the bug. For high priority issues, especially ones that do not take too much time (< 1 day effort), please create the issue and pull it into the sprint to be worked on.
+- If the secondary on-call engineer is new, please pair-program to help them onboard onto the process. Include them when you do deployment and point out various issues to watch out for (i.e. incoming bugs, Sentry, etc.) so that they get an understanding of what the role entails.
+- Watch the [#single-cell-ops](https://czi-sci.slack.com/archives/C0244PQK934), [#single-cell-eng](https://czi-sci.slack.com/archives/C023Q1APASK), and [#single-cell-data-wrangling](https://czi-sci.slack.com/archives/C024HCSH9PT) Slack channels for any questions or issues reported by any engineers, PM, or our Curation team. Please triage these issues, notify the right person, and if appropriate file an issue to keep track of the bug. For high priority issues, especially ones that do not take too much time (< 1 day effort), please create the issue and pull it into the sprint to be worked on.
```

Comment:
I'd delete "especially ones that do not take too much time (< 1 day effort)". I think we can be much more strict about the issues we pull in during on-call to avoid too much distraction with ongoing mission work.

Reply:
Yeah, I think we also should define "high priority issue".  How about:
`Please triage these issues, notify the right person(s), and if actionable, file an issue to keep track of the bug. If PMs deem the the issue to be _high priority_,  please pull the issue into the sprint to be worked on by the on-call engineer(s).`